olive/workflows/snpe/convertquantize/convertquantize.py

```diff
@@ -44,10 +44,10 @@ def convertquantize(
     model_file = Path(model).resolve()
     if model_file.suffix == ".onnx":
         logger.info("Loading model...")
-        model = ONNXModel(model_file, name)
+        model = ONNXModel(model_file)
     elif model_file.suffix == ".pb":
         logger.info("Loading model...")
-        model = TensorFlowModel(model_file, name)
+        model = TensorFlowModel(model_file)
     else:
         raise Exception(f"Unsupported model format: {model_file.suffix}")
```

Comment:
raise ValueError?

Reply:
Will address in the future when we refactor the other exceptions in Olive.